packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -363,7 +401,18 @@ class AnimationController extends Animation<double>
     return _animateToInternal(target, duration: duration, curve: curve);
   }
 
-  TickerFuture _animateToInternal(double target, { Duration duration, Curve curve = Curves.linear }) {
+  TickerFuture _animateToInternal(double target, { Duration duration, Curve curve = Curves.linear, AnimationBehavior animationBehavior }) {
+    final AnimationBehavior behavior = animationBehavior ?? this.animationBehavior;
+    double scale = 1.0;
+    if (ui.window.accessibilityFeatures.disableAnimations) {
+      switch (behavior) {
+        case AnimationBehavior.normal:
+          scale = 0.05;
```

Comment:
The descriptions of `AnimationBehavior.normal` might as well point out the similarity with timeDilation=0.05 as well as explaining what the actual effect is in, uh, layman's terms.